web/src/main/webapp/WEB-INF/pages/candidates.jsp

```diff
@@ -51,7 +51,7 @@
                             </th>
                             <th data-field="phone" data-align="left" data-sortable="true"><spring:message
                                     code="welcome.candidates.table.phone.title"/></th>
-                            <th data-field="positions" data-align="left" data-sortable="true" data-formatter="positionFormatter">
```

Comment:
I like this data-formatter, what was wrong with it?

Without this the position names are concatenated without space.